backoffice/src/Models/ViewModels/ImageView.cs

```diff
@@ -0,0 +1,22 @@
+using Droits.Models.Entities;
+
+namespace Droits.Models.ViewModels;
+
+public class ImageView : BaseEntityView
+{
+    public ImageView()
+    {
+        
+    }
+
+
+    public ImageView(Image image) : base(image)
+    {
+        Id = image.Id;
```

Comment:
Might be nice to have the Created date in here so you can display "Uploaded date" on the image view or something